eq-author-api/schema/resolvers/routing2/routingRule2/index.js

```diff
@@ -15,25 +25,103 @@ Resolvers.RoutingRule2 = {
   expressionGroup: routingRule => routingRule.expressionGroup,
   routing: ({ id }, args, ctx) => {
     const pages = flatMap(section => section.pages, ctx.questionnaire.sections);
-    return find(page => {
-      if (page.routing && some({ id }, page.routing.rules)) {
-        return page.routing;
+    const allRouting = flatMap(page => page.routing, pages);
+    const routing = find(routing => {
+      if (some({ id }, routing.rules)) {
+        return routing;
       }
-    }, pages);
+    }, allRouting);
+    return routing;
   },
 };
 
 Resolvers.Mutation = {
-  createRoutingRule2: async (root, { input }, ctx) =>
-    ctx.modifiers.RoutingRule.create(input.routingId),
-  updateRoutingRule2: async (root, { input: { id, destination } }, ctx) => {
+  createRoutingRule2: async (root, { input }, ctx) => {
+    const routing = find(
+      { id: input.routingId },
+      flatMap(
+        page => page.routing,
+        flatMap(section => section.pages, ctx.questionnaire.sections)
+      )
+    );
+
+    const routingRule = createRoutingRule({
+      expressionGroup: createExpressionGroup({
+        expressions: [
+          createExpression({
+            left: createLeftSide({
+              type: "Null",
+              nullReason: "NoRoutableAnswerOnPage",
```

Comment:
Needs to default to the first routable answer on page, falling back to this if there isn't one.